router-tests/automatic_persisted_queries_test.go

```diff
@@ -33,7 +33,7 @@ func TestAutomaticPersistedQueries(t *testing.T) {
 				res := xEnv.MakeGraphQLRequestOK(testenv.GraphQLRequest{
 					Extensions: []byte(`{"persistedQuery": {"version": 1, "sha256Hash": "does-not-exist"}}`),
 				})
-				require.Equal(t, `{"errors":[{"message":"persisted query not found","extensions":{"code":"PERSISTED_QUERY_NOT_FOUND"}}]}`, res.Body)
+				require.Equal(t, `{"errors":[{"message":"PersistedQueryNotFound","extensions":{"code":"PERSISTED_QUERY_NOT_FOUND"}}]}`, res.Body)
```

Comment:
Why? Motivation? Please add a PR description.

Reply:
Apollo client is reliant on the error _message_ not the error code.

Reply:
Thanks